src/Workspaces/Core/Portable/Workspace/Solution/AdditionalDocumentState.cs

```diff
@@ -14,7 +14,7 @@ internal sealed class AdditionalDocumentState : TextDocumentState
         private readonly AdditionalText _additionalText;
 
         private AdditionalDocumentState(
-            SolutionServices solutionServices,
+            HostWorkspaceServices solutionServices,
```

Comment:
Consider changing `solutionServices` to `services`, given that `SolutionServices` no longer exsits as well as for consistency. (there's many across this PR)

Reply:
I considered that, but i actually liked the name as tying this to the Solution-snapshot is hte idea this is trying to convey.

Reply:
In that case, maybe use `solutionServices` consistenly? :)

Reply:
Good suggestion. For now i'd like to keep things very mechanical.  There will be a bunch of PRs for this effort, so this streamlines things.  :)